src/components/navbar/navbar.js

```diff
@@ -9,21 +9,21 @@ export const Navbar=()=>{
     }
 
     return (
-        <header class="e-com-header">
-        <nav class="e-com-navbar shop-nav ecom-bg-blue">
+        <header className="e-com-header">
+        <nav className="e-com-navbar shop-nav ecom-bg-blue">
             <Link to="/">
-                <div class="ecom-white">showTube</div>
+                <div className="ecom-white">showTube</div>
             </Link>
-            <ul class="e-com-nav-items">
+            <ul className="e-com-nav-items">
```

Comment:
you can change class name as it is not "e-com" looks odd for reviewer

Reply:
ok will do it